discord/member.py

```diff
@@ -402,6 +402,22 @@ def permissions_in(self, channel):
         """
         return channel.permissions_for(self)
 
+    def has_role(self, role_id):
+        """Check if a user has a role with the given ID.
+
+        Parameters
+        ----------
+        role_id: :class:`int`
+            The ID for the role.
+
+        Returns
+        -------
+        bool
+            :code:`True` if the user has a role with the
```

Comment:
And again, it's a member.
```suggestion
            :code:`True` if the member has a role with the
```